GameServer/Logic/Content/ContentManager.cs

```diff
@@ -159,17 +159,17 @@ public string GetUnitStatPath(string model)
 
         public string GetSpellDataPath(string spellName)
         {
-            try
-            {
-                var contentType = "Spells";
-                var contentPackages = _content[contentType][spellName];
-                var fileName = string.Format("{0}/{1}.json", spellName, spellName);
-                return GetContentPath(contentPackages, contentType, fileName);
-            }
-            catch
+            var contentType = "Spells";
+
+            if (!_content.ContainsKey(contentType) || !_content[contentType].ContainsKey(spellName))
             {
                 return "";
```

Comment:
I still think this should raise a custom exception like "ContentNotFoundException", or at least log an error of that type if not raise it (This means you'll have to create a new class with that name extending the base Exception class).